app/controllers/concerns/blacklight/bookmarks.rb

```diff
@@ -81,7 +81,7 @@ def create
     current_or_guest_user.save! unless current_or_guest_user.persisted?
 
     success = @bookmarks.all? do |bookmark|
```

Comment:
Will the `all?` short-circuit as soon as one bookmark fails to validate? Is the intent to persist what we can and drop any failures on the floor?

I'm not sure where we're creating multiple bookmarks at once either.. but it goes way, way back.

Reply:
Good questions -- I think the block would stop as soon as it tries to persist something invalid.  Would it be better not to add any at all if one or more are invalid?